README.md

```diff
@@ -20,7 +20,7 @@ docker build \
 
 # Optionally override the tag for the base `tailscale/tailscale` image
 docker build \
-  --build-arg TAILSCALE_TAG=v1.29.18 \
+  --build-arg TAILSCALE_TAG=v1.34.0 \
```

Comment:
As of right now 1.34.2 is the latest release of Tailscale (but this is the current state internally)